### DISCUSSION TITLE
Auto-import using POST documentation

### DISCUSSION THREAD

**Discussion by spasche** [firefly-iii/firefly-iii#7890] [Firefly III]
Hello,

I followed [the documentation](https://docs.firefly-iii.org/data-importer/advanced/automation/#automated-imports-using-the-web-post) to setup auto-import using POST.

All went well, but I noticed something unexpected in the documentation, in particular:
> Personal Access Token required
> The POST commands only work when you're using a Personal Access Token to authenticate.

Is this still true? I didn't use an access token to call the endpoint and it still worked.

More specifically, I didn't pass the `--header 'Authorization: Bearer ey....'` argument to curl which is specified in the examples (only the `secret` GET parameter).

So I was wondering if the documentation might be out of date or if a personal acccess token is supposed to be used (in which case I guess that would be a bug if it works without).

Thanks

**Comment by JC5** [comment]
But do you have the access token in your `.env` file?

**Reply by spasche** [reply]
Yes, I do (I have `FIREFLY_III_ACCESS_TOKEN` set in the environment, I'm using the Docker image). Ok, so now I understand why I don't need to specify it in the POST request.
Thanks for the clarification, maybe this could be specified in the documentation that it can either be specified in the POST requests or otherwise be set in the importer environment with `FIREFLY_III_ACCESS_TOKEN`.

**Reply by JC5** [reply]
Will do!
